wings/market/coinbase_pro_market.pyx

```diff
@@ -480,6 +461,122 @@ cdef class CoinbaseProMarket(MarketBase):
+
+                        if tracked_order.is_buy:
+                            self.logger().info(f"The market buy order {tracked_order.client_order_id} has completed "
+                                               f"according to Coinbase Pro user stream.")
+                            self.c_trigger_event(self.MARKET_BUY_ORDER_COMPLETED_EVENT_TAG,
+                                                 BuyOrderCompletedEvent(self._current_timestamp,
+                                                                        tracked_order.client_order_id,
+                                                                        tracked_order.base_asset,
+                                                                        tracked_order.quote_asset,
+                                                                        (tracked_order.fee_asset
+                                                                         or tracked_order.base_asset),
+                                                                        float(tracked_order.executed_amount),
+                                                                        float(tracked_order.quote_asset_amount),
+                                                                        float(tracked_order.fee_paid)))
+                        else:
+                            self.logger().info(f"The market sell order {tracked_order.client_order_id} has completed "
+                                               f"according to Coinbase Pro user stream.")
+                            self.c_trigger_event(self.MARKET_SELL_ORDER_COMPLETED_EVENT_TAG,
+                                                 SellOrderCompletedEvent(self._current_timestamp,
+                                                                         tracked_order.client_order_id,
+                                                                         tracked_order.base_asset,
+                                                                         tracked_order.quote_asset,
+                                                                         (tracked_order.fee_asset
+                                                                          or tracked_order.quote_asset),
+                                                                         float(tracked_order.executed_amount),
+                                                                         float(tracked_order.quote_asset_amount),
+                                                                         float(tracked_order.fee_paid)))
+                    else: # reason == "canceled":
+                        execute_amount_diff = 0
+                        tracked_order.last_state = "canceled"
+                        self.c_trigger_event(self.MARKET_ORDER_CANCELLED_EVENT_TAG,
+                            OrderCancelledEvent(self._current_timestamp, tracked_order.client_order_id))
+                        execute_amount_diff = 0
```

Comment:
should stop tracking the order if done message is received

Reply:
added